deutsches-archaologisches-institut.csl

```diff
@@ -1,20 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<style class="in-text" version="1.0" delimiter-precedes-et-al="never" initialize-with="." demote-non-dropping-particle="sort-only" default-locale="de" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
@bellerophons-pegasus, can you change `default-locale="de"` to `default-locale="de-DE"`? (that way the style explicitly uses the "de-DE" CSL locale file from https://github.com/citation-style-language/locales/)